crate_universe/src/context/crate_context.rs

```diff
@@ -28,6 +28,12 @@ pub struct CrateDependency {
 #[serde(default)]
 pub struct TargetAttributes {
     /// The module name of the crate (notably, not the package name).
+    ///
+    /// This must be the first field of `TargetAttributes` to make it the
```

Comment:
Small nit but I don't think this information is ideal for docs. Feels like just a code comment. Do you know of a good way to not have this show up in rustdoc renderings?

Reply:
We can use `//` comments.